cluster/saltbase/salt/kubelet/default

```diff
@@ -15,6 +15,15 @@
   {% set api_servers = "--api_servers=https://" + ips[0][0] + ":6443" -%}
 {% endif -%}
 
+# Disable registration for the kubelet running on the master on GCE.
```

Comment:
It keeps things consistent with the current deployment. I'd be happy to change this in a future PR to be configurable, but didn't want too many moving pieces here. For one, we'd need to change cluster validation to expect N+1 nodes (rather than N) since the master is now counted. And I didn't want to worry about other ramifications at the same time as changing the node creation pattern.